orca/service_test.go

```diff
@@ -131,6 +130,8 @@ func (s) TestE2E_CustomBackendMetrics_OutOfBand(t *testing.T) {
 		errCh <- nil
 	}()
 
+	wg.Wait()
```

Comment:
We don't need this waitGroup, there is already a for loop which Wait for the server to push metrics which indicate the completion of all the unary RPCs made from the above goroutine.